openmmtools/alchemy.py

```diff
@@ -1143,6 +1426,39 @@ def _build_alchemical_bond_list(alchemical_atoms, reference_forces, system):
     # Internal usage: Alchemical forces
     # -------------------------------------------------------------------------
 
+    def _add_alchemical_forces(self, alchemical_system, alchemical_forces_by_lambda):
```

Comment:
Is it possible to fall back on not using force groups if there are not enough available?

Reply:
You can set `split_alchemical_forces` to `False` if there are not enough force groups available. Otherwise, `RuntimeError` is raised.

Reply:
Right, but I mean can we just default to fall back to non-`split_alchemical_forces` if there are too many forces to split into the 32 groups?

Reply:
Ah! I see, sorry. I went for the exception because in general, I would prefer to know if there's a performance penalty instead of proceeding silently, but I can change this behavior if you think we should automatically fall back to `split_alchemical_forces=False`.

I think it'd be very easy for an application to go with something like this anyway:
```python
try:
    factory.create_alchemical_system(...)
except RuntimeError:
    factory.split_alchemical_forces = False
    factory.create_alchemical_system(...)
```